src/Mod/TechDraw/Gui/Resources/TechDraw.qrc

```diff
@@ -113,6 +113,8 @@
         <file>icons/actions/section-down.svg</file>
         <file>icons/actions/section-left.svg</file>
         <file>icons/actions/section-right.svg</file>
+        <file>icons/actions/techdraw-MoveView.svg</file>
+        <file>icons/actions/techdraw-CopyView.svg</file>
```

Comment:
@WandererFan do you mind renaming the SVGs in this naming format: TechDraw-{Move|Copy}View.svg